frontend/components/auction/debt/DebtAuctionTransactionFlow.vue

```diff
@@ -142,6 +156,10 @@ export default Vue.extend({
             type: Boolean,
             default: true,
         },
+        isFlopperAuthorized: {
```

Comment:
As requested before, please don't use maker terms in our codebase:
```suggestion
        isDebtAuctionAuthorized: {
```
Here and everywhere else!